__tests__/app.test.js

```diff
@@ -341,4 +341,56 @@ describe('GET /api/users', ()=>{
                 })
             })
     })
+})
+describe('FEATURE: GET /api/articles (queries)', () =>{
+    test('200: should respond with an array of the articles filtered by a specific topic', () => { 
+        return request(app)
+        .get('/api/articles?topic=mitch&sort_by=created_at&order=desc')
+        .expect(200)
+        .then(({ body }) => {
+        expect(body.articles.length).toBeGreaterThan(0),
+        expect(body.articles[0].topic).toEqual('mitch')
```

Comment:
Ideally we want to be checking every element of the array not just the 0th element as it might be a coincidence that this has the correct topic.